### PR TITLE
switch per-backup/restore logs to use logrus

### LINKED ISSUE #114: Convert backup logs to use logrus

### LINKED ISSUE #115: Convert restore logs to use logrus

### PR DESCRIPTION
**Pull request by skriss**
Fixes #114 
Fixes #115 

(to be merged after #122)

### CODE REVIEW

File: pkg/backup/backup.go [hunk 1]
```diff
@@ -133,13 +133,11 @@ func getNamespaceIncludesExcludes(backup *api.Backup) *collections.IncludesExclu
 }
 
 type logger struct {
```
Comment (davecheney) [inline pkg/backup/backup.go]: Does this provide much value? Why not just use the logrus logger directly?

File: pkg/restore/restore.go [hunk 2]
```diff
@@ -223,13 +225,11 @@ func (kr *kubernetesRestorer) Restore(restore *api.Restore, backup *api.Backup,
...
 type logger struct {
-	w io.Writer
+	impl *logrus.Logger
 }
 
 func (l *logger) log(msg string, args ...interface{}) {
-	// TODO use a real logger that supports writing to files
-	now := time.Now().Format(time.RFC3339)
-	fmt.Fprintf(l.w, now+" "+msg+"\n", args...)
+	l.impl.Infof(msg, args...)
```
Comment (davecheney) [inline pkg/restore/restore.go]: same question as above; is there value in adding an abstraction above logrus?

File: pkg/backup/backup.go (hunk 1 above)
Comment (skriss) [reply pkg/backup/backup.go]: @davecheney no, was just taking the smallest possible step towards logrus here. Plan is to get rid of this intermediate layer.

**Comment by davecheney** [comment]
SGTM. Make it so.

**Pull request merged**